File: 13 taksi_udruzenje/taksi_udruzenje.py
import sys
brMin, Min = [int(i) for i in input().split()]
for ulaz in sys.stdin:
    brPotencijalnogMin, minDoDolaska = [int(i) for i in ulaz.split()]
    if minDoDolaska < Min:
        brMin = brPotencijalnogMin
        Min = minDoDolaska
print(int(brMin))

File: 13 taksi_udruzenje/taksi_udruzenje.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace taksi_udruzenje
{
    class Program
    {
        static void Main(string[] args)
        {
            string ulaz;
            string[] podaci;
            int brMin;
            int Min;
            ulaz = Console.ReadLine();
            podaci = ulaz.Split();
            brMin = int.Parse(podaci[0]);
            Min = int.Parse(podaci[1]);
            while ((ulaz = Console.ReadLine()) != null) {
                podaci = ulaz.Split();
                int minDoDolaska = int.Parse(podaci[1]);
                if (minDoDolaska < Min)
                {
                    brMin = int.Parse(podaci[0]);
                    Min = minDoDolaska;
                }
            }
            Console.WriteLine(brMin);
        }
    }
}
